app/Helpers/Configuration.php

```diff
@@ -53,6 +53,28 @@ public function set(string $key, string $value, string $profile = 'default'): vo
         $this->config = $config;
     }
 
+    public function getCommandConfiguration(string $command, string $profile = 'default'): array
+    {
+        $this->config = $this->readConfig();
+
+        $command = trim($command);
+
+        return Arr::get($this->config, "{$profile}.command_options.{$command}", []);
+    }
+
+    public function setCommandConfiguration(string $command, string $key, string $value, string $profile = 'default'): void
+    {
+        $config = $this->config;
+
+        $command = trim($command);
+
+        Arr::set($config, "{$profile}.command_options.{$command}.{$key}", $value);
+
+        file_put_contents($this->configFilePath(), json_encode($config, JSON_PRETTY_PRINT));
+
+        $this->config = $config;
+    }
```

Comment:
There's some repetition here. Can we not call `get` and `set` from within these methods?